.github/workflows/post-publish-update-version.yml

```diff
@@ -0,0 +1,21 @@
+name: Update application version after release has been published
+
+on:
+  push:
+    branches:
+      - 'update-ga'
+
+env:
+  GH_TOKEN: ${{ secrets.GH_RELEASE_DEPLOYMENT_TOKEN }}
```

Comment:
does something use this token? if not, there's an embedded token that Github Actions provides that can be used automatically for thing like CLI access via `gh`

Reply:
Not sure if / whether it is used.  Only reference to it in the Frame repo is in these actions files

Reply:
well it's definitely not used in the current version of this script so we can get rid of it for now